test/jdk/java/lang/System/SecurityManagerWarnings.java

```diff
@@ -25,6 +25,7 @@
  * @test
  * @bug 8266459 8268349 8269543 8270380
  * @summary check various warnings
+ * @requires !java.enablePreview
```

Comment:
What it the reason that this. test failures with --enable-preview?

Reply:
Ough, this test puzzled me. It fails with --enable-preview. However, I think I need more time to investigate the issue. So I update PR to exclude test that explicitly says that it shouldn't be executed with --enable-preview.

Reply:
It ran ok for me, once I got the command line flags correct.
It ran ok if I added `@enablePreview`.

It also ran ok with an explicit \@run command:  (it does not currently have an \@run command).
```
 * @run main/othervm --enable-preview SecurityManagerWarnings
 ```

Reply:
For me it fails with
----------System.err:(18/917)----------
 stdout: [];
 stderr: [Error: Unable to initialize main class SecurityManagerWarnings
Caused by: java.lang.NoClassDefFoundError: jdk/test/lib/process/OutputAnalyzer
]
 exitValue = 1
that seems pretty strange, might be test library issue?

Reply:
I haven't been able to reproduce that locally. Even with mis-matched compilation of the test library and test code.

I noticed the NoClassDefFoundError message comes from the child process. 
The child is invoked with test.noclasspath=true and no path to the test library.  (intentionally)
The SecurityManagerWarning class explicitly refers to OutputAnalyzer.
There might be a path in which the new VM tries to load the OutputAnalyzer (and throw an error)

Finding a way to reproduce locally might be necessary to track down the cause.
Perhaps adding -Xlog (for the child) might provider more information about the sequence of events.

Reply:
Please file a separate bug for the failure, so further investigation can be done.